src/components/datetime/datetime.ts

```diff
@@ -619,28 +619,14 @@ export class DateTime extends Ion implements AfterContentInit, ControlValueAcces
       }
     }
 
-    // default to assuming this month has 31 days
```

Comment:
Can you explain the change a little more? why does it make a difference moving this piece of code?